packages/rest-explorer/templates/index.html.ejs

```diff
@@ -5,9 +5,9 @@
 <head>
   <meta charset="UTF-8">
   <title>LoopBack API Explorer</title>
-  <link rel="stylesheet" type="text/css" href="./swagger-ui.css">
-  <link rel="icon" type="image/png" href="./favicon-32x32.png" sizes="32x32" />
-  <link rel="icon" type="image/png" href="./favicon-16x16.png" sizes="16x16" />
+  <link rel="stylesheet" type="text/css" href="<%- explorerPath %>/swagger-ui.css">
```

Comment:
Are you sure this is going to work when the app is running behind a reverse proxy like nginx?

Reply:
It's the same deal with or without this change. See https://github.com/strongloop/loopback-next/pull/2293.